index.html

```diff
@@ -1,21 +1,83 @@
-    <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <body>
+    <header>
+      <nav>
+        <img src="img/karma-logo.svg" alt="company logo" class="logo" />
+        <img src="img/menu-hamburger.svg" alt="" class="hamburger" />
+        <ul class="nav-list">
+          <li><a href="#" class="nav-item first-item">Meet Karma</a></li>
+          <li><a href="#" class="nav-item">How it Works</a></li>
+          <li><a href="#" class="nav-item">Store</a></li>
+          <li><a href="#" class="nav-item">Blog</a></li>
+          <li><a href="#" class="nav-item">Help</a></li>
+          <li><a href="#" class="nav-item">Login</a></li>
+        </ul>
+      </nav>
+    </header>
 
-</body>
+    <main>
+      <section class="hero">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <button>Learn More</button>
+      </section>
+
+      <section class="about">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="flex-container">
+          <article class="flex-item">
+            <img src="img/icon-devices.svg" alt="devices" />
+            <p>Internet for all devices</p>
+          </article>
+          <article class="flex-item">
+            <img src="img/icon-coffee.svg" alt="cup of coffee" />
+            <p>Boost your productivity</p>
+          </article>
+          <article class="flex-item">
+            <img src="img/icon-refill.svg" alt="refill" />
+            <p>Pay as You Go</p>
+          </article>
+        </div>
+        <hr />
+      </section>
+    </main>
+
+    <footer>
+      <p>Join us on</p>
+      <section class="social">
+        <div class="img-container">
+          <img src="./img/twitter-icon.svg" alt="twitter icon" />
+        </div>
+        <div class="img-container">
+          <img src="./img/facebook-icon.svg" alt="facebook icon" />
+        </div>
+        <div class="img-container">
+          <img src="./img/instagram-icon.svg" alt="instagram icon" />
+        </div>
+      </section>
```

Comment:
This could potentially be a list

Reply:
Wasn't sure which tag i should use here so decided to use div. But list makes much more sense! Will change that too :)